$OpenBSD: patch-kcalcore_calformat_cpp,v 1.4 2014/07/09 20:03:30 zhuk Exp $
Allow UUID library to be found even if not in default search path.
--- kcalcore/calformat.cpp.orig	Thu Jan  2 23:32:09 2014
+++ kcalcore/calformat.cpp	Mon Jan 13 16:22:42 2014
@@ -33,7 +33,7 @@
 #include "calformat.h"
 #include "exceptions.h"
 
-#if defined(HAVE_UUID_UUID_H)
+#if UUID_FOUND
 #include <uuid/uuid.h>
 #else
 #include <KRandom>
@@ -122,7 +122,7 @@ void CalFormat::setLoadedProductId(const QString &id)
 
 QString CalFormat::createUniqueId()
 {
-#if defined(HAVE_UUID_UUID_H)
+#if defined(UUID_FOUND)
     uuid_t uuid;
     char suuid[64];
 
